lib/reline/key_stroke.rb

```diff
@@ -67,9 +90,9 @@ def match_unknown_escape_sequence(input, vi_mode: false)
       idx += 1
     else
       # `ESC char` or `ESC ESC char`
-      return :unmatched if vi_mode
+      return UNMATCHED if vi_mode
     end
-    input[idx + 1] ? :unmatched : input[idx] ? :matched : :matching
+    input[idx + 1] ? UNMATCHED : input[idx] ? MATCHED : MATCHING
```

Comment:
nit: can we please expand this, my brain isn't good enough to process nested ternary operators 😅